pyproject.toml

```
@@ -20,7 +20,7 @@ classifiers = [
     "Development Status :: 7 - Inactive",
 ]
 dependencies = [
-    "numpy>=1.16",
+    "numpy",
```

```suggestion
    "numpy">=1.22,
```

To be consistent with NEP-29 (https://numpy.org/neps/nep-0029-deprecation_policy.html).